ddtrace/internal/debug.py

```diff
@@ -99,7 +99,7 @@ def collect(tracer):
             integration_configs[module] = "N/A"
 
     pip_version = packages_available.get("pip", "N/A")
-
+    
```

Comment:
this change seems unrelated, please be sure to run `black` on any files you change.

`riot run -s fmt`

Reply:
Thanks! I ran this is the latest set of commit and I think this got removed :)